styles.css

```diff
@@ -1,786 +1,786 @@
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.h-contact {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  text-align: center;
+  line-height: 52px;
+  color: #fff;
+  padding: auto 2.4rem;
+  display: flex;
+  flex-direction: column;
+  gap: 5.2rem;
+}
+
+.para {
+  padding-top: 1%;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  color: #fff;
+  line-height: 28px;
+  text-align: center;
+  margin-top: 1%;
+}
+
+input {
+  border: none;
+  border-radius: 10px 5pt;
+  display: block;
+  margin-top: 4%;
+  margin-bottom: 4%;
+  width: 32.7rem;
+  height: 4.5rem;
+  padding: 1.5rem 3.8rem 1.5rem 1.6rem;
+}
+
+input::placeholder {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #172b4d;
+}
+
+fieldset {
+  border: none;
+  padding: 2rem auto;
+}
+
+textarea {
+  border: none;
+  text-align: left;
+  width: 32.7rem;
+  height: 17.5rem;
+  color: #172b4d;
+  font-size: 1.5rem;
+  border-radius: 0.8rem;
+  padding: 1.5rem 3.8rem 1.5rem 1.6rem;
+}
+
+textarea::placeholder {
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  margin-top: 2%;
+  color: #b3bac5;
+  font-size: 1.5rem;
+}
+
+.l-btn {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+  background: #fff;
+  height: 50px;
+  padding-left: 3%;
+  margin-left: 3%;
+}
+
+.resume {
+  color: #6070ff;
+  background-color: #fff;
+  font-size: 1.7rem;
+  font-weight: 500;
+  padding: 1.2rem 1.6rem;
+  border: none;
+  border-radius: 0.8rem;
+  cursor: pointer;
+}
+
+.cont-btn {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: #6070ff;
+  background-color: #fff;
+  font-size: 1.7rem;
+  font-weight: 500;
+  padding: 1.2rem 1.6rem;
+  border: none;
+  border-radius: 0.8rem;
+  cursor: pointer;
+  margin: 0 auto;
+  margin-top: 5%;
+}
+
+.logo-mobile {
+  gap: 3%;
+}
+
+.abt-self {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.about-me {
+  background: #ffff;
+}
+
+.abt-p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  padding-top: 4%;
+  padding-bottom: 4%;
+}
+
+.languages-inputs-mobile {
+  flex-direction: co;
+}
+
+.frontend {
+  display: none;
+}
+
+.sep1 {
+  margin-bottom: 0;
+  width: 95%;
+  color: #dfe1e6;
+}
+
+.sep2 {
+  margin-bottom: 12%;
+  width: 95%;
+  color: #dfe1e6;
+  padding-bottom: 35%;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.r-a {
+  padding-right: 5%;
+  padding-bottom: 5%;
+}
+
+@media screen and (min-width: 768px) {
+  .container {
+    margin: auto 10%;
+  }
+
+  .contact-me {
+    background-image: url('./images/contact-original.png');
+    background-repeat: no-repeat;
+    background-size: 90%;
+    background-position: 90px 90px !important;
+  }
+

```

Comment:
- [OPTIONAL] The proper background image does not reflect on your contact me section on the mobile version of your project. Kindly look into it and fix it.

### Your Design
![your-mobile-contact-me](https://github.com/hetrox8/portfolio1/assets/26597432/1d7dc22b-e106-472e-b45c-0fc3aecf9317)

![your-mobile-contact-me-2](https://github.com/hetrox8/portfolio1/assets/26597432/2c256e5e-7b50-4ba7-9f49-1ac9d3a8c2b7)

[your-contact-mobile.webm](https://github.com/hetrox8/portfolio1/assets/26597432/424abc03-4846-42bd-8ee9-5f7c7b9f9c2f)


### Figma Design 
![figma-contact-me-mobile](https://github.com/hetrox8/portfolio1/assets/26597432/60c43ab0-8dd8-4b68-9fe2-3e1e0a075972)